recipes-core/lcd-tools/lcd-tools/lcd-tools.cpp

```diff
@@ -0,0 +1,23 @@
+#include "lcd-tools.h"
+
+#include <cxxopts.hpp>
+
+using namespace AsteroidOS::LCD_Tools;
+
+int main(int argc, char** argv){
+	cxxopts::Options options("lcd-tools", "Various controls for the TicWatch LCD");
```

Comment:
What other options do you intend to add in the future ?
For now this feels rather unnecessary because the tool only has three modes:
- No arguments are passed and it just doesn't do anything
- --sync-time is provided and it syncs time, cool
- --help is provided and it tells you to use --sync-time

At the moment it feels like this entire function could be a `return SyncTime();`

Reply:
Indeed it will include different options like `sync-steps`, etc.